README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
```

Comment:
This text is about navigation (how to play), should be emphasized in bold or italic
Lines 4 and 5 is just general info
Line 6 should be moved up to line 4 (also about how to play)